stardog/admin.py

```diff
@@ -429,7 +431,12 @@ def virtual_graphs(self):
         """
         r = self.client.get("/admin/virtual_graphs")
         virtual_graphs = r.json()["virtual_graphs"]
-        return list(map(lambda name: VirtualGraph(name, self.client), virtual_graphs))
+        return list(
+            map(
+                lambda name: VirtualGraph(name.replace("virtual://", ""), self.client),
```

Comment:
This might be a breaking change for some users that are already replacing this.

Reply:
This is done before the object are created so a valid object get created. 

A user may perform a similar approach in their code which would be replacing the `self.graph_name` attribute with a fixed one. If they using the following approach to fix the object, they would probably be doing the following 

`vg.graph_name = vg.graph_name.replace("virtual://","")`

Which would fix the object.  Should they run this logic on a valid object, they would still be ok.

Reply:
Should be noted that the object created today does not work.